Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutAnalytics.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+class BTPayPalNativeCheckoutAnalytics {
+    
+    // MARK: - Tokenize Events
+    
+    static let tokenizeStarted = "paypal-native:tokenize:started"
+    static let tokenizeFailed = "paypal-native:tokenize:failed"
+    static let tokenizeSucceeded = "paypal-native:tokenize:succeeded"
+    static let tokenizeCanceled = "paypal-native:tokenize:canceled"
+    
+    // MARK: - Order Creation Fail
```

Comment:
We should keep the marks consistent with this file in the other modules.

Maybe 

```
// MARK: - Conversion Events
...
// MARK: - Additional Detail Events
```